gamemode/index.js

```diff
@@ -336,18 +332,50 @@ sdk.on("chatMessage", (player, message) => {
 sdk.on("chatCommand", (player, message, command, args) => {
     console.log(`[GAMEMODE] Player ${player.getNickname()} used: ${command}. (${message})`);
 
-    switch (command) {
-        case 'home':
-            player.setPosition(sdk.Vector3(-989.397, -289.772, 2.805));
-            player.sendChat(`[SERVER] Teleported to home!`)
-            break
+    if (command === "showArgs") {
+        sdk.Chat.sendToAll(`[SERVER] Player ${player.getNickname()} used /showArgs with args: ${args}`);
+        return;
+    }
 
-        case 'showArgs':
-            sdk.Chat.sendToAll(`[SERVER] Player ${player.getNickname()} used /showArgs with args: ${args}`);
-            break
+    if (command === "heal") {
+        player.setHealth(100);
+        player.sendChat(`[SERVER] Health restored!`);
+        return;
+    }
 
-        default:
-            sdk.Chat.sendToPlayer(player, `[SERVER] Unknown command "${command}"`);
-            break
+    if (command === "home") {
+        player.setPosition(sdk.Vector3(-989.397, -289.772, 2.805));
+        player.sendChat(`[SERVER] Teleported to home!`);
+        return;
     }
+
+    if (command === "veh") {
+        // const modelName = args[0] ?? "berkley_810"; // doesn't works well yet
+        const modelName = "berkley_810";
+        const veh = sdk.World.createVehicle(modelName);
+
+        if (veh) {
+            // doesn't work yet, still an object even if the vehicle doesn't exist
+            veh.setPosition(player.getPosition());
+            veh.setRotation(player.getRotation());
+
+            player.sendChat(`[SERVER] ${modelName} successfully created!`);
+            return;
+        }
+
+        player.sendChat(`[SERVER] Unable to create vehicle ${modelName}!`);
+        return;
+    }
+
+    if (command === "wep") {
+        // doesn't works yet
+
+        // const weaponId = parseInt(args[0], 10) ?? 85;
+        const weaponId = 85;
+        player.addWeapon(weaponId, 200);
+        player.sendChat(`[SERVER] Weapon received!`);
+        return;
+    }
+
+    sdk.Chat.sendToPlayer(player, `[SERVER] Unknown command "${command}"`);
```

Comment:
Please avoid rewriting code styles. Keep switch.

Reply:
It was me who transformed it into a switch and it was a mistake, that's why I went back to early-return

Reply:
Switch or map is way cleaner than if-forest, at scale. You can go back to it then

Reply:
I don't agree with this one...

I tried go back to switch and it makes the code much less readable, especially for complex commands. (veh / wep)

Switching to a map and/or dedicated functions per command seems a bit overkill to me at this level...

@hurfy any opinion on it?

Reply:
Scoped switches sound like a decent compromise, example:
```js
switch (val) { 
  case "foo": { 
    bar(); 
  } break;

  case "bar": { 
    bar(); 
    baz();
  } break; 

  default: { 
    foo(); 
  } break; 
}
```

Reply:
My point is: at scale (meaning that the default gamemode will contain more and more commands by the time), an if-forest will become a nightmare, where a switch appears to be the correct decision.

Even better would be a map of commands, where the key is the string command, and the value the handler.

Reply:
What is your POV on this new approach?

https://github.com/MafiaHub/MafiaMP/pull/47/commits/90713d44e2662902f8ad9702b496ad2dd1984b82

Reply:
I like the idea. Commands should for sure be in a separate file!